tpkg/ede.tdir/ede.help

```diff
@@ -0,0 +1 @@
+Test if NSD recognizes ascii NSID in config.
```

Comment:
This text should talk about the EDE option instead of the NSID option.  It is also unused by our current set up; then also remove it from ede.dsc if you decide to remove the file.

Reply:
resolved in commit fcafc1a